src/treetensornetworks/solvers/dmrg.jl

```diff
@@ -1,58 +1,34 @@
-function eigsolve_solver(;
-  solver_which_eigenvalue=:SR,
-  ishermitian=true,
-  solver_tol=1e-14,
-  solver_krylovdim=3,
-  solver_maxiter=1,
-  solver_verbosity=0,
-)
-  function solver(H, init; normalize=nothing, region=nothing, half_sweep=nothing)
-    howmany = 1
-    which = solver_which_eigenvalue
-    vals, vecs, info = eigsolve(
-      H,
-      init,
-      howmany,
-      which;
-      ishermitian,
-      tol=solver_tol,
-      krylovdim=solver_krylovdim,
-      maxiter=solver_maxiter,
-      verbosity=solver_verbosity,
-    )
-    psi = vecs[1]
-    return psi, (; solver_info=info, energies=vals)
-  end
-  return solver
-end
-
 """
 Overload of `ITensors.dmrg`.
 """
+
+function dmrg_sweep(
```

Comment:
Maybe we should rename to something more descriptive, like `dmrg_sweep_updates` or `dmrg_sweep_plan` (same for renaming `tdvp_sweep`).

I think I like `dmrg_sweep_plan` since it indicates it is a plan for what to do during the sweep, rather than actually performing the sweep/updates.